docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-json-api-convention-integration-guide.md

```diff
@@ -0,0 +1,502 @@
+    {
+        return [
+            new CustomRouteRouterPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Glue\GlueStorefrontApiApplicationExtension\Dependency\Plugin\RouteMatcherPluginInterface>
+     */
+    protected function getRouteMatcherPlugins(): array
+    {
+        return [
+            new CustomRouteRouteMatcherPlugin(),
+            new ResourceRouteMatcherPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceInterface>
+     */
+    protected function getResourcePlugins(): array
+    {
+        return [
+            new DummyResource(),
+        ];
+    }
+
+    /**
+     * @return \Spryker\Glue\GlueStorefrontApiApplicationExtension\Dependency\Plugin\RequestResourceFilterPluginInterface
+     */
+    public function getRequestResourceFilterPlugin(): RequestResourceFilterPluginInterface
+    {
+        return new RequestResourceFilterPlugin();
+    }
+
+    /**
+     * @return array<\Spryker\Glue\GlueStorefrontApiApplicationExtension\Dependency\Plugin\RouteProviderPluginInterface>
+     */
+    protected function getRouteProviderPlugins(): array
+    {
+        return [
+            new DummyRouteProviderPlugin(),
+        ];
+    }
+}
+
+```
+
+**src/Pyz/Glue/GlueStorefrontApiApplicationGlueJsonApiConventionConnector/GlueStorefrontApiApplicationGlueJsonApiConventionConnectorDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Glue\GlueStorefrontApiApplicationGlueJsonApiConventionConnector;
+
+use Spryker\Glue\DummyModuleApi\Plugin\DummyResourceRelationshipPlugin;
+use Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\ResourceRelationshipCollectionInterface;
+use Spryker\Glue\GlueStorefrontApiApplicationGlueJsonApiConventionConnector\GlueStorefrontApiApplicationGlueJsonApiConventionConnectorDependencyProvider as SprykerGlueStorefrontApiApplicationGlueJsonApiConventionConnectorDependencyProvider;
+
+class GlueStorefrontApiApplicationGlueJsonApiConventionConnectorDependencyProvider extends SprykerGlueStorefrontApiApplicationGlueJsonApiConventionConnectorDependencyProvider
+{
+    /**
+     * @param \Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\ResourceRelationshipCollectionInterface $resourceRelationshipCollection
+     *
+     * @return \Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\ResourceRelationshipCollectionInterface
+     */
+    protected function getResourceRelationshipPlugins(
+        ResourceRelationshipCollectionInterface $resourceRelationshipCollection
+    ): ResourceRelationshipCollectionInterface {
+        $resourceRelationshipCollection->addRelationship(
+          'dummyResourceType',
+          new DummyResourceRelationshipPlugin()
+        );
+
+        return $resourceRelationshipCollection;
+    }
+}
+```
+
+{% info_block warningBox “Verification” %}
+
+If everything is set up correctly, you should be able to access `http://glue-storefront.de.spryker.local/stores`.
```

Comment:
What about verification for the pagination/sorting/filtering parameters?
There should be usage and verification for includes. 
There should be verification for custom route we created.

The general rule is - you need to have a verification step for EACH plugin that is being installed. In our case, some of them will not be verifiable without each other - we can combine them into one verifacation.